### PR TITLE
Dependencies update & small fixes

### PR DESCRIPTION
**Pull request by Drapegnik**
* migrate to [`gulp@4`](https://github.com/gulpjs/gulp/releases/tag/v4.0.0) - please check that everything works fine
* update to [`stylelint@9`](https://github.com/stylelint/stylelint/releases/tag/9.2.1)
* update to [`bulma@0.6.2`](https://github.com/jgthms/bulma/releases) - please check out release notes for new features

* add unpublished article tile
* add publication date
* add edit button

screen shot 2018-06-08 at 18 29 40
screen shot 2018-06-08 at 18 29 19

> P.S: please help we with this blue color, I updated bulma and something broke

### CODE REVIEW

**Comment by Drapegnik** [comment]
[an 'unpublished' mark on tiles so that it is clear for admins which articles are available for them only](https://trello.com/c/XrkpJadk/116-an-unpublished-mark-on-tiles-so-that-it-is-clear-for-admins-which-articles-are-available-for-them-only)

**Comment by Drapegnik** [comment]
[Article tile: add publishing date and edit button](https://trello.com/c/7pgDujac/160-article-tile-add-publishing-date-and-edit-button)

File: gulpfile.js [hunk 1]
```diff
@@ -85,24 +86,26 @@ gulp.task('fa:fonts', () =>
...
-    .pipe(ejs(templateVariables, {}, { ext: '.html' }).on('error', gutil.log))
+    .pipe(ejs(templateVariables, {}, { ext: '.html' }).on('error', log))
     .pipe(gulp.dest(config.buildPath))
 );
 
-gulp.task('html:lint', ['ejs:compile'], () =>
+const lintHtml = () =>
   gulp
     .src(`${config.buildPath}/**/*.html`)
+    // .pipe(plumber())
```
Comment (tyndria) [inline gulpfile.js]: why do you comment plumber here?

**Comment by tyndria** [comment]
Sometimes it's not enough space for publication date
untitled

**Comment by Drapegnik** [comment]
@UladBohdan 👍

**Pull request merged**